edgelet/edgelet-http/src/unix.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Why are these changes required?

Reply:
Oops.. need to rebase, I suppose.. certainly not part of my changes